lib/ansible/module_utils/eos.py

```diff
@@ -180,6 +186,17 @@ def _load_params(self):
             if key in NET_COMMON_ARGS.keys():
                 if not params.get(key) and value is not None:
                     params[key] = value
+
+        env_overrides = dict(
+            username='ANSIBLE_NET_USERNAME',
+            password='ANSIBLE_NET_PASSWORD',
+            ssh_keyfile='ANSIBLE_NET_SSH_KEYFILE',
+            authorize='ANSIBLE_NET_AUTHORIZE',
+            auth_pass='ANSIBLE_NET_AUTH_PASS',
+        )
+        for key, env_var in env_overrides.items():
+            if env_var in os.environ:
+                params[key] = os.environ[env_var]
```

Comment:
For people following at home, we had a talk with privateip, qalthos, bcoca, jimi-c, and myself about this on slack, decided for now to have parameters overriding env vars.  We'll discuss this more next week and see if we can push the values somewhere else (perhaps into inventory variables that are set on the connection plugin).